### ISSUE TITLE
[Deploy] Google Voice

### ISSUE THREAD

**Issue by BehindTheMath** [meetfranz/plugins#87]
### Service
Google Voice

### Link to your repository
https://github.com/BehindTheMath/franz-recipe-google-voice

### Additional Information
This is an update from the [legacy plugin](https://github.com/meetfranz/plugins-legacy/tree/develop/googlevoice), to one compatible with Franz 5.

**Comment by BehindTheMath** [comment]
I pushed another commit that should fix an issue with package.json that was mentioned on Slack.

@adlk Are there any other issues that are preventing this from being added?

**Comment by simonkotwicz** [comment]
@BehindTheMath maybe the icons should be updated to the new google voice icon?

**Comment by BehindTheMath** [comment]
@simonkotwicz I can't find an SVG for the new icon.

**Comment by BehindTheMath** [comment]
>@simonkotwicz I can't find an SVG for the new icon.

Never mind. I drew it myself.

Both the PNG and SVG icons have been updated.

**Comment by eddiezane** [comment]
Is this waiting on anything?

**Comment by BehindTheMath** [comment]
@eddiezane Nothing that I know of.

@adlk @haraldox Can you clarify this?

**Comment by Zarel** [comment]
I tried to use this, and it isn't working on the latest version of Google Voice.

I screwed around with `webview.js` a bit and this makes it work:

https://github.com/BehindTheMath/franz-recipe-google-voice/pull/2 (merged)

**Comment by drewrisinger** [comment]
What's the status of this? Any chance of being merged into Franz anytime soon?

**Comment by BehindTheMath** [comment]
@adlk Is there any reason this hasn't been merged yet?

**Comment by drewrisinger** [comment]
FWIW, I've been testing the GVoice plugin for the last 2-ish weeks, and it has been running smoothly @adlk

**Comment by adlk** [comment]
Deployed ✅
